.eslintrc.js

```diff
@@ -37,6 +37,7 @@ module.exports = {
                 ignorePackages: true,
             },
         ],
+        "jest/no-focused-tests": "error",
```

Comment:
Ah, interesting!

Reply:
We use this rule in `webapp` too.  Will be nice to move some of this stuff into our shared eslint config so that we don't have to remember to add this to every project.